README.md

```diff
@@ -31,24 +32,30 @@ use pip:
 pip install pasqal-cloud
 ```
 
-If you wish to **install the development version of the pasqal-cloud from source** instead, do the following from within
+Likewise, with Python 3.9.0 or higher, `pulser-pasqal` is installed with:
```

Comment:
it's 3.8 in `setup.py` but I guess that's because `pulser-core` supports only greater than 3.9 since v1